gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -1215,15 +1215,23 @@ function BeginRound()
 
 	local plys = player.GetAll()
 
+	local activeCount = 0
+
 	for i = 1, #plys do
 		local ply = plys[i]
 
 		ply:ResetRoundDeathCounter()
 
+		local isActive = ply:Alive() and ply:IsTerror()
+
 		-- a player should be considered "was active in round" if they received a role
-		ply:SetActiveInRound(ply:Alive() and ply:IsTerror())
+		ply:SetActiveInRound(isActive)
+
+		activeCount = isActive and activeCount+1 or activeCount
```

Comment:
Did you test this? I'm pretty sure lua fails with the `+1` and the `or` here, you should add brackets.

On a different note, please always add `spaces` around the `+` sign.

Reply:
I did test it and it worked like intendet. I will add spaces.